tests/dialogs/dialogs-list.directive.spec.js

```diff
@@ -0,0 +1,86 @@
+describe('app.components.DialogsListDirective', function() {
+  var $scope;
+  var $compile;
+  var element;
+
+  beforeEach(function () {
+    module('app.services', 'app.config', 'app.states', 'app.components', 'gettext');
+    bard.inject('DialogsState', '$state', 'Session', '$httpBackend', '$timeout');
+  });
+
+  beforeEach(inject(function (_$compile_, _$rootScope_, _$document_) {
+    $compile = _$compile_;
+    $scope = _$rootScope_;
+    $document = _$document_;
+  }));
+
+  var compileHTML = function (markup, scope) {
+    element = angular.element(markup);
+    $compile(element)(scope);
+
+    scope.$digest();
+  };
+
+  beforeEach(function () {
+    Session.create({
+      auth_token: 'b10ee568ac7b5d4efbc09a6b62cb99b8',
+    });
+    $httpBackend.whenGET('').respond(200);
+
+    $scope.dialogs = [
+      {
```

Comment:
Note:  We started adding this mock data to the 'tests/mock' directory as seperate json files.